### ISSUE TITLE
Variables in document template doesn't work

### ISSUE THREAD

**Issue by yegle** [google/exposure-notifications-verification-server#537]
### TL;DR
We uses variables in the alert.

https://github.com/google/exposure-notifications-verification-server/blob/f13ac85195f11be1a37f1dd2cf1508704f849065/terraform/alerting/monitoring.tf#L108

The actual delivered alert contains the variables as-is without substitution.

**Expected behavior**
https://cloud.google.com/monitoring/alerts/doc-variables doc works.

**Observed behavior**
Variables appear as-is (i.e. `${resource.label.host} request throttled by ratelimit middleware`)

### Reproduction

1.  Went to ...
2.  Ran ...
3.  Clicked on ...

**Screenshots**

**Environment**

- OS:
- Browser:
- Version:

**Additional information**
I've confirmed with `gcloud alpha monitoring policies list` that the `$$` in the alert policy is properly escaped.

**Comment by yegle** [comment]
After looking into this further I believe this is an issue with the delivery method we chose for our internal delivery. The notification from Cloud Monitoring (e.g. as email) does correctly handled the variable substitution.
